client/landing/gutenboarding/onboarding-block/plans/index.tsx

```diff
@@ -91,7 +91,7 @@ const PlansStep: React.FunctionComponent< Props > = ( { isModal } ) => {
 				<Title>{ __( 'Select a plan' ) }</Title>
 				<SubTitle>
 					{ __(
-						'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 30 days.'
+						'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 14 days.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

Reply:
Is it worth making the number of days a variable with sprintf (or whatever the i18n equivalent is)? That would at least prevent future changes here from requiring another round of translations.

Reply:
I actually just dropped that exact commit. 

There's some Slack conversation around when to use the abstracted variable (p1610550051436500-slack-C02AED43D). If I understand correctly, we'd only do this if the number itself is variable in this particular instance (i.e. 7 vs 14, based on plan length, [like it is here](https://github.com/Automattic/wp-calypso/pull/48872/files#diff-010394ebaf934e04dc170d0355b1146369fa4d5db7daa5bb6895eb13cb8219f8R217)). This boils down to there being the possibility of a single string being translated differently in a single language based on the value of that number.

@akirk or @yuliyan might be able to correct me if I'm wrong.

Reply:
I don't want to force my opinion, but I think that it's not unlikely that these periods may need to be changed again in the future, so replacing them with variables now would make it easier for future updates with no need for re-translations.

Reply:
@yuliyan thanks for weighing in! You're not forcing your opinion - you're the expert. I think I misunderstood the slack thread. I made the change in 7e142fea35de36c9a412280921b5dd4f23109d75 and 8670723182e792f3be04a08a7abad962572d0003.